components/scripts/gradle/01-validate-incremental-building.sh

```diff
@@ -149,12 +149,10 @@ execute_build() {
 
 print_gradle_command() {
```

Comment:
I suggest to use the same if/else construct with the almost-duplicated command as in the other experiments.